source/reference/methods.txt

```diff
@@ -837,6 +837,18 @@ Native Methods
 
      - Description
 
+   * - :manual:`load() </reference/method/load/>`
+
+     - Loads and runs a JavaScript file in the shell.
+
+       In ``mongosh``, scripts loaded with the :method:`load()` method
+       support the ``__filename`` and ``__dirname`` Node.js objects.
```

Comment:
It might be a bit confusing to refer to these as objects, because they are strings and in JS terminology objects are a distinct type from strings? But I guess anyone familiar with Node.js would be familiar with that as well.

What might be worth pointing out is that these are always absolute paths.

Reply:
Thank you for the comments! I called them "objects" initially because I saw them listed here, [as nodejs Global Objects](https://www.tutorialspoint.com/nodejs/nodejs_global_objects.htm). However, it seems like "environment variables" might be more appropriate.

Reply:
Maybe we could call them `per-script variables`? That might still not be ideal, but given how `environment variables` also has a very specific meaning, I think calling them something like that might be better.

Reply:
Ah, I'd gleaned from [this tutorial](https://www.digitalocean.com/community/tutorials/nodejs-how-to-use__dirname) that they might be environment variables, but happy to use different terminology if "environment variables" is not appropriate.

I'm inclined to just generalize to say  "__filename and __dirname variables", since I worry that the "per-script" terminology might be a little confusing.